docs/design/FeatureSwitches.md

```diff
@@ -0,0 +1,13 @@
+# Feature Switches
+
+Certain features of MAUI can be enabled or disabled using feature switches. The easiest way to control the features is by putting the corresponding MSBuild property into the app's .csproj file. Disabling unnecessary features can help reducing the app size when combined with the [`full` timming mode](https://learn.microsoft.com/en-us/dotnet/core/deploying/trimming/trimming-options).
+
+| MSBuild Property Name | AppContext Setting | Description |
+|-|-|-|
+| XamlLoadingIsEnabled | Microsoft.Maui.RuntimeFeature.IsXamlLoadingEnabled | When disabled, all XAML loading at runtime will throw an exception. This will affect usage of APIs such as the `LoadFromXaml` extension method. This feature can be safely turned off when all XAML resources are compiled using XamlC (see [XAML compilation](https://learn.microsoft.com/en-us/dotnet/maui/xaml/xamlc)). This feature is enabled by default for all configurations except for NativeAOT. |
```

Comment:
From meeting:

MSBuild: `$(MauiXamlLoadingSupport)`
Flag: `Microsoft.Maui.RuntimeFeature.XamlLoadingSupported` - Microsoft.Maui.dll / Core.csproj, internal class

Reply:
Should the flag be `XamlRuntimeParsingSupported` or `*Is*XamlRuntimeParsingSupported`? The `Is-` prefix would follow the same pattern as `DynamicCodeSupport -> IsDynamicCodeSupported` and other runtime flags.